### ISSUE TITLE
Analyzers ambassador

### ISSUE THREAD

**Issue by qemm2** [exakat/exakat#108]
Hi, I'm trying to modify the Ambassador theme cause I want to remove the compatibility analyzers.
Can You Help me?

I read about the excluded analyzers but i can't change that theme.

Thanks in advance

**Comment by exakat** [comment]
Ambassador or Diplomat is a report that has a fixed set of requirements. They are not configurable, as they are now. Excluding analyzers will be overwritten by the report's requirements.

The simplest option is to open the produced report, and remove the mention of the 'compatibility' in the menu. 

Are you trying to speed up the audit process?

**Comment by qemm2** [comment]
Yes, I need do a report and I thought that we can reuse the ambassador code only change the analyzers. 
I need to add more analyzers and remove anothers.
For example , removing the next false postives like the next one because we don't want the compability, and I thought that if we  can deactivate the Compability analyzers we can continue with the report. 
"
PHP 7.0 Scalar Typehints | All, CompatibilityPHP53, CompatibilityPHP54, CompatibilityPHP55, CompatibilityPHP56 | 51 | 35 | Critical | 31 %
-- | -- | -- | -- | -- | --"

**Comment by qemm2** [comment]
How can I overwrite the analyzers? I'have comment the themes in the exakat.ini but it doesn't works

**Comment by exakat** [comment]
So, Ambassador or Diplomat have a fixed set of prerequisite. Requesting the report means those pre-requisite will be added to the audit. 

Currently, the best option is to subclass Ambassador in the source code, strip the compatibility sections, and use other rulesets.

**Comment by philippechevieux** [comment]
Will it be possible in a future version to have a customizable Ambassador report ? Or at least choose in Ambassador what Analyzer to display or not ? 

Like @qemm2 i do have some analyzer with a lot of false positives.
